src/etc/modules

```diff
@@ -16,6 +16,11 @@ virtio_scsi
 virtio-gpu
 virtio-rng
 
+## Vmware
+vmw_pvscsi
+vmxnet3
+mptspi
```

Comment:
Testing this on a Ubuntu 22.04 node, together with dependencies this adds about 400kB of additional modules, which IMO is quite a lot, given that this is adding support for what I consider to be a dying proprietary platform, I'm not sure that this is worth it.

Reply:
Fair enough.